tests/chains/fabric/chaincode/fabibc/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/datachainlab/ibc-examples/chaincode
+module github.com/hyperledger-labs/yui-relayer/tests/chains/fabric/chaincode/fabibc
```

Comment:
Is this related to this PR?

Reply:
@siburu 
Not directly related, but it's a minor fix and I want to include it.